dev/tab1tha/main.py

```diff
@@ -0,0 +1,31 @@
+import pandas as pd
+import sklearn
+import seaborn as sns
+import k_nn
+from sklearn.model_selection import train_test_split
+
+#import data into a pandas dataframe
+df = pd.read_csv('..\..\datasets\generated.csv')
+
+#EDA exploratory analysis of the data
+#Exploration
+print('*'*50+'HEAD'+'*'*50)
+print(df.head())
+print('*'*50+'DESCRIBE'+'*'*50)
+print(df.describe())
+print('*'*50+'INFO'+'*'*50)
+print(df.info())
+print('*'*100)
+
+#todo visualisation
+
+data = df.drop('label', axis = 1)
+target = df['label']
+#split data set into train and test sets 
+data_train, data_test, target_train, target_test = train_test_split(data,target, test_size = 0.30, random_state = 10, stratify = target)
```

Comment:
What is the motivation for a specific split ratio of exactly `test_size = 0.30`?
Does performance vary when this ratio changes?
Please motivate the choice of this parameter or provide a sub-analysis that explores the effects of this parameter.

Reply:
The default value is 0.25, I chose 0.30 as a close approximate. I have investigated the effect of test size on performance in #43.